.vscode/extensions.json

```diff
@@ -2,6 +2,7 @@
   "recommendations": [
     "rvest.vs-code-prettier-eslint",
     "streetsidesoftware.code-spell-checker",
-    "DigitalBrainstem.javascript-ejs-support"
+    "DigitalBrainstem.javascript-ejs-support",
+    "bradlc.vscode-tailwindcss"
```

Comment:
Adding [Tailwind CSS IntelliSense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) as a recommended plugin provides linting and autocomplete when writing tailwind classnames


Uploading intellisense720.mov…